deposit/src/test/java/edu/unc/lib/deposit/normalize/Proquest2N3BagJobTest.java

```diff
@@ -148,9 +144,6 @@ public void testWithAttachments() {
 	}
 
 	private void testWithAttachments(Model model, Resource primaryResource) {
```

Comment:
This test is failing currently.  I would suggest finding everything in these jobs that reference ContentModelHelper and trying to replace them with the RDF helpers, since the former is intended for use with fcrepo3.  In this test it is checking for a defaultWebObject, which is superseded by Cdr.primaryObject.  In the job there are a number of cases where it is adding older content models like AGGREGATE_WORK

 Inside the Proquest normalization job there is still a reference to defaultWebObject, which is from ContentModelHelper.  One of our goals is to not be using anything from that class anymore, since all of the properties have migrated to the RDF helpers.